### PR TITLE
chore(events): add event data to auctions

### PR DESCRIPTION
**Pull request by dtfiedler**

### CODE REVIEW

File: src/main.lua [hunk 1]
```diff
@@ -2695,71 +2725,70 @@ addEventingHandler("auctionBid", utils.hasMatchingTag("Action", ActionMap.Auctio
...
 		return
 	end
 
-	local status, auctionBidOrError =
-		pcall(arns.submitAuctionBid, name, bidAmount, bidder, timestamp, processId, type, years)
-	if not status then
+	local shouldContinue2, result = eventingPcall(msg.ioEvent, function(error)
 		ao.send({
 			Target = msg.From,
-			Action = "Invalid-" .. ActionMap.AuctionBid .. "-Notice",
```
Comment (arielmelendez) [inline src/main.lua:2726]: Is this the incorrect way to do this or is there just an advantage to setting these to Tags manually?

File: src/main.lua [hunk 2]
```diff
@@ -138,6 +138,25 @@ local function addRecordResultFields(ioEvent, result)
 	end
 end
 
+local function addAuctionResultFields(ioEvent, result)
+	ioEvent:addFieldsIfExist(result, {
+		"bidAmount",
+		"bidder",
```
Comment (arielmelendez) [inline src/main.lua:144]: Will this always be the same as the from field? If so we can likely omit this one.

File: src/main.lua [hunk 3]
```diff
@@ -138,6 +138,25 @@ local function addRecordResultFields(ioEvent, result)
...
+		"rewardForInitiator",
+		"rewardForProtocol",
+		-- TODO: we could return some computed values of the bid relative to start and end price
```
Comment (arielmelendez) [inline src/main.lua:147]: Is it hard to get the start and end price values of the auction? I think it would be very valuable data to have here.

**Review by arielmelendez — DISMISSED** [review]

File: src/main.lua (hunk 1 above)
Comment (dtfiedler) [reply src/main.lua:2726]: we have some incosistency accross notices that this is addressing

File: src/main.lua (hunk 3 above)
Comment (dtfiedler) [reply src/main.lua:147]: we compute it based on the bid terms (years, and type) - we need to bubble it up from other places

File: src/main.lua (hunk 2 above)
Comment (dtfiedler) [reply src/main.lua:144]: it will, can remove

File: src/main.lua (hunk 3 above)
Comment (dtfiedler) [reply src/main.lua]: added here - [5931635](https://github.com/ar-io/ar-io-network-process/pull/154/commits/593163517498d51631ceb6c39cd5f54592398589)

**Pull request merged**